draft-ietf-tls-tls13.md

```diff
@@ -3518,9 +3520,23 @@ other than for computing other secrets.)
 ##  MTI Cipher Suites
 
 In the absence of an application profile standard specifying otherwise, a
-TLS-compliant application MUST implement the cipher suite
-[TODO:Needs to be selected](https://github.com/tlswg/tls13-spec/issues/32).
-(See {{cipher-suites}} for the definition.)
+TLS-compliant application MUST implement the following cipher suites:
+
+~~~~
+    TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 
+    TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+~~~~
+
+These cipher suites MUST support both digital signatures and key agreement
```

Comment:
"key agreement" -> "key exchange"

This document uses "key exchange" almost everywhere. Might even be a good idea to change the 2 instances of "agreement" to "exchange" to be consistent.

To be explicitly clear, it could be: "ECDHE key exchange"